docs/source/llms/langchain/index.rst

```diff
@@ -461,3 +461,77 @@ The agent can be loaded and used for inference as follows:
         ]
     }
     agent.invoke(query)
+
+How can I evaluate a LangGraph Agent?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `mlflow.evaluate <https://mlflow.org/docs/latest/model-evaluation/index.html>`_ function provides 
+a robust way to evaluate model performance. 
+
+LangGraph agents, especially those with chat functionality, can return multiple messages in one 
+inference call. Given ``mlflow.evaluate`` performs naive comparisons between raw predictions and a specified
+ground truth value, it is the user's responsibility to reconcile potential differences prediction output
+and ground truth.
+
+Often, the best approach is to use a `custom function <https://mlflow.org/docs/latest/llms/llm-evaluate/index.html#evaluating-with-a-custom-function>`_ 
+to process the response. Below we provide an example of a custom function that extracts the last chat 
+message from a LangGraph model. This function is then used in mlflow.evaluate to return a single 
+string response, which can be compared to the `"ground_truth"` column.
+
+.. code-block:: python
+
+    # Note that we assume the `model_uri` variable is present
+    loaded_model = mlflow.langchain.load_model(model_uri)
```

Comment:
nit: When using custom function, we actually don't need to log a model

Reply:
Good point but I think this is a more realistic user journey. We're looking to demo evaluate on langgraph, which for most cases, would involve logging a model. Will add a note that logging the model is optional for evaluation